pkg/annotation/directive.go

```diff
@@ -87,7 +87,7 @@ func (d Directives) Int(key string) (int, bool) {
 func (d Directives) Object(key string) Directives {
 	v, ok := d[key]
 	if !ok {
-		return make(Directives)
+		return nil
 	}
 	m, ok := v.(map[string]interface{})
 	if !ok {
```

Comment:
The following line should also be `nil`, right?

Reply:
if it exists but theres nothing in the map, then we should likely keep it? I pushed this already but can make changes if we need

Reply:
the other question is does anything outside of env vars use this today?

Reply:
yeah that maybe we would want to error and say that its set to an incorrect type even, should i add a card for this? I dont think weve really touched the logic of directives too much. Were unblocked for now but youre right we may want to add more to this